src/js/_enqueues/wp/theme.js

```diff
@@ -1132,6 +1132,12 @@ themes.view.Themes = wp.Backbone.View.extend({
 		});
 	},
 
+	renderThemeCount: function( count ) {
+		this.liveThemeCount = count;
+		var currentLocale = document.documentElement.lang;
```

Comment:
Can this be const instead?

Reply:
Looks like it can, the esversion changed earlier this year to allow it: https://core.trac.wordpress.org/changeset/57702